jupyterhub/notebook-images/overlays/additional/generic-data-science-notebook-imagestream.yaml

```
@@ -13,26 +14,37 @@ spec:
   lookupPolicy:
     local: true
   tags:
-  # N Version of the image (v2-2023a-20230526-c4c062e)
-  - annotations:
-      opendatahub.io/notebook-software: '[{"name":"Python","version":"v3.9"}]'
-      opendatahub.io/notebook-python-dependencies: '[{"name":"Boto3","version":"1.26"},{"name":"Kafka-Python","version":"2.0"},{"name":"Kfp-tekton","version":"1.5"},{"name":"Matplotlib","version":"3.6"},{"name":"Numpy","version":"1.24"},{"name":"Pandas","version":"1.5"},{"name":"Scikit-learn","version":"1.2"},{"name":"Scipy","version":"1.10"},{"name":"Elyra,"version":"3.15"}]'
-      openshift.io/imported-from: quay.io/modh/odh-generic-data-science-notebook
-      opendatahub.io/workbench-image-recommended: 'true'
-    from:
-      kind: DockerImage
-      name: $(odh-generic-data-science-notebook-image-n)
-    name: "2023.1"
-    referencePolicy:
-      type: Source
-  # N-1 Version of the image
-  - annotations:
-      opendatahub.io/notebook-software: '[{"name":"Python","version":"v3.8"}]'
-      opendatahub.io/notebook-python-dependencies: '[{"name":"Boto3","version":"1.17"},{"name":"Kafka-Python","version":"2.0"},{"name":"Matplotlib","version":"3.4"},{"name":"Numpy","version":"1.19"},{"name":"Pandas","version":"1.2"},{"name":"Scikit-learn","version":"0.24"},{"name":"Scipy","version":"1.6"}]'
-      openshift.io/imported-from: quay.io/modh/odh-generic-data-science-notebook
-    from:
-      kind: DockerImage
-      name: $(odh-generic-data-science-notebook-image-n-1)
-    name: "1.2"
-    referencePolicy:
-      type: Source
+    # N Version of the image
+    - annotations:
+        opendatahub.io/notebook-software: '[{"name":"Python","version":"v3.9"}]'
+        opendatahub.io/notebook-python-dependencies: '[{"name":"Boto3","version":"1.26"},{"name":"Kafka-Python","version":"2.0"},{"name":"Kfp-tekton","version":"1.5"},{"name":"Matplotlib","version":"3.6"},{"name":"Numpy","version":"1.24"},{"name":"Pandas","version":"1.5"},{"name":"Scikit-learn","version":"1.2"},{"name":"Scipy","version":"1.10"},{"name":"Elyra","version":"3.15"}]'
```

We need to update the versions of the packages on the N version in all the ImageStreams, like:
```suggestion
              opendatahub.io/notebook-python-dependencies: '[{"name":"Boto3","version":"1.28"},{"name":"Kafka-Python","version":"2.0"},{"name":"Kfp-tekton","version":"1.5"},{"name":"Matplotlib","version":"3.6"},{"name":"Numpy","version":"1.24"},{"name":"Pandas","version":"1.5"},{"name":"Scikit-learn","version":"1.3"},{"name":"Scipy","version":"1.11"},{"name":"Elyra","version":"3.15"},{"name":"PyMongo","version":"4.5"},{"name":"pyodbc","version":"4.0"}, {"name":"Codeflare-SDK","version":"0.8"}, {"name":"sklearn-onnx","version":"1.15"}, {"name":"Psycopg","version":"3.1"}, {"name":"MySQL Connector/Python","version":"8.0"}]'
```